stdlib/public/core/CharacterProperties.swift

```diff
@@ -23,11 +23,19 @@ extension Character {
   }
 
   /// A Boolean value indicating whether this is an ASCII character.
-  @inlinable
+  @_alwaysEmitIntoClient @_silgen_name("inlineIsASCII")
```

Comment:
Why are we doing these symbol shenanigans? Someone calling this symbol can just call into the new implementation no? Can't we just leave these `@inlinable` and introduce the new outline functions as UFI and remove the duplicated small check in there?

Reply:
The "new" outline functions also provide the old implementation for existing binaries that didn't inline them, so they have to support all cases.

Reply:
Nvm ignore me, the new outline functions don't exist on older platforms.

Reply:
Can we name this symbol something like `swift_stdlib_inlineIsASCII`? Similarly for the ascii value one

Reply:
The new outline functions do "exist" on older platforms because they're mangled the same as the old implementations; so on older platforms we get the inlining speedup for "normal" characters and codesize win, but zalgotext will run at the old speed.

Reply:
I'm saying if we didn't do the symbol shenanigans then the outlined functions don't exist because they'd have new symbols. With the shenanigans they do exist because it's just the older symbol.

Reply:
libswiftCore.dylib currently exports a property descriptor for this property. Is it okay to get rid of it?

If we must abuse `@_silgen_name`, then I think we should at least use a unique enough symbol name, like `@Azoy` suggested.

Reply:
How much of a difference it would make if we simply lost the `@inlinable`, rather than engaging in these `@_silgen_name` hacks?

Reply:
Quite a lot. The old implementations never should have been inlinable, but the new implementations really want to be, as they're only a few instructions.